package.json

```diff
@@ -23,8 +23,10 @@
   "dependencies": {
     "@(-.-)/env": "0.3.2",
     "@prisma/client": "4.14.1",
+    "@types/parse-duration": "0.3.0",
```

Comment:
I think `parse-duration` already have type built-in.
![Untitled1](https://github.com/kaogeek/kaogeek-discord-bot/assets/1791353/4298a3cf-439e-48b0-ac81-b861cf9ffcb2)

If it absolutely have to, you should add to `devDependencies` becasuse it doesn't need in runtime.